### PR TITLE
Update SXG doc per desktop and `Vary: Cookie`.

### PR DESCRIPTION
**Pull request by twifkak**
Both features are now launched.

### CODE REVIEW

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ subhead: |
   origin of a resource independently of how it was delivered.
 authors:
   - katiehempenius
+  - twifkak
 date: 2020-10-14
-updated: 2021-05-12
+updated: 2022-06-07
```
Comment (malchata) [inline src/site/content/en/fast/signed-exchanges/index.md:11]:
```suggestion
updated: 2022-06-09
```

**Review by malchata — APPROVED** [review]
LGTM, updated update date.

**Comment by twifkak** [comment]
Thanks! Please merge when ready.

**Pull request merged**